templates/ruby/pkgs/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, bundlerEnv, ruby }:
+let
+  # the magic which will include gemset.nix
+  gems = bundlerEnv {
+    name = "my-package-env";
+    inherit ruby;
+    gemfile = ../Gemfile;
+    lockfile = ../Gemfile.lock;
+    gemset = ../gemset.nix;
```

Comment:
I think you said you used bundix? Will that be commented or documented somewhere?

Reply:
@mkenigs yes for sure. I'll make special mention of it + the steps to follow in the docs. I'll actually add a corresponding comment here too. Thanks!

Reply:
pushed a change now to resolve

Reply:
Hmm just seeing the formatting change, did you add something for this?

Reply:
Still need to add the comment on but fix. Was referring to formatting

Reply:
Comment added